.drone.yaml

```diff
@@ -21,7 +21,7 @@ steps:
       - tag
 
 - name: build
-  image: rancher/dapper:v0.4.1
+  image: rancher/dapper:v0.5.8
```

Comment:
any chance we could please bump this up to v0.6.0? I have already upgraded this in https://github.com/harvester/seeder/pull/9

Reply:
Yes, updated it. Thanks!